app/views/layouts/application.html.haml

```diff
@@ -18,8 +18,7 @@
         = ' - '
       = title
 
-    = stylesheet_pack_tag 'common', media: 'all'
-    = stylesheet_pack_tag stylesheet_for_layout, media: 'all'
```

Comment:
The perf benefit of the `common` CSS file is definitely minimal, but why remove it?